### ISSUE TITLE
how to solve training align problem .

### ISSUE THREAD

**Issue by leoluopy** [InternLM/xtuner#194]
after fine tune i got the log below:
 [20][2/2]  lr: 1.9581e-04  eta: 0:18:21  time: 3.0714  data_time: 0.0069  memory: 10314  grad_norm: 0.0218  loss: 0.0003

loss is so small , but the inference output is still very different from my SFT ground truth .  do any one has some idea ?

**Comment by LZHgrla** [comment]
How many samples for your custom dataset?

**Comment by leoluopy** [comment]
full set has about 1000 QA pairs , but now i try to use only 3 QA pairs to finetune ,  then problem is that loss is so small , but the inference output is still very different from my SFT ground truth( 3 question answer pairs) .   i am not sure if there is some thine wrong with my step to step fine tune training operations . give me some ideas  ?? @LZHgrla   , thanks .

**Comment by LZHgrla** [comment]
Hi @leoluopy 

Try to use full 1000 QA pairs and set `pack_to_max_length` to `False` in config. Also, try to use a larger learning rate.

**Comment by LZHgrla** [comment]
Closing this issue, feel free to reopen if you have more questions.
